src/api/params/params.spec.js

```diff
@@ -49,64 +56,90 @@ describe('api.v2.params', () => {
   describe('extractTrendsParams', () => {
     beforeEach(() => {
       fixtureStore.trends = {
+        ...fixtureStore.trends,
         lens: 'ItemName',
         trendDepth: 10,
       };
-      fixtureStore.query = {
-        dateInterval: 'quantum',
-      };
+      fixtureStore.query.dateInterval = 'quantum';
     });
 
     it('gets trends params', () => {
-      actual = sut.extractTrendsParams(fixtureStore);
+      const { filters, query, trends } = fixtureStore;
+      actual = sut.extractTrendsParams(filters, query, trends);
       expect(actual).toEqual({
+        chartType: 'line',
+        date_received_max: '2018-01-01',
+        date_received_min: '2011-07-21',
+        field: 'all',
         lens: 'itemname',
+        no_aggs: true,
         trend_depth: 10,
         trend_interval: 'quantum',
+        searchField: 'all',
+        size: 0,
+        sub_lens: 'sub_product',
+        reducerValues: {
+          focus: '',
+          lens: 'ItemName',
+          subLens: 'sub_product',
+          trendDepth: 10,
+        },
       });
     });
 
     it('handles subLens and focus item', () => {
       fixtureStore.trends.focus = 'Focus item';
       fixtureStore.trends.subLens = 'Sub-issue';
-      actual = sut.extractTrendsParams(fixtureStore);
+      const { filters, query, trends } = fixtureStore;
+      actual = sut.extractTrendsParams(filters, query, trends);
       expect(actual).toEqual({
+        chartType: 'line',
+        date_received_max: '2018-01-01',
+        date_received_min: '2011-07-21',
+        field: 'all',
         focus: 'Focus item',
         lens: 'itemname',
+        no_aggs: true,
+        reducerValues: {
+          focus: 'Focus item',
+          lens: 'ItemName',
+          subLens: 'Sub-issue',
+          trendDepth: 10,
+        },
+        searchField: 'all',
+        size: 0,
         sub_lens: 'sub_issue',
         trend_depth: 10,
         trend_interval: 'quantum',
       });
     });
   });
 
-  describe('parseParams', () => {
+  describe('parseParamsToQuery', () => {
     beforeEach(() => {
       fixtureStore.query.searchText = 'foo';
       fixtureStore.trends.lens = 'Issue';
       fixtureStore.trends.subLens = 'Sub-issue';
     });
 
-    describe('parseParamsToQuery', () => {
-      it('handles bogus searchFieldMap', () => {
-        fixtureStore.query.searchField = 'Bogus value';
-        const actual = sut.extractQueryParams(fixtureStore.query);
-        expect(actual.field).toEqual('all');
-      });
+    it('handles bogus searchFieldMap', () => {
+      fixtureStore.query.searchField = 'Bogus value';
+      const actual = sut.extractQueryParams(fixtureStore.query);
+      expect(actual.field).toEqual('all');
```

Comment:
I would steer clear of using `toEqual()` clauses for evaluating primitive values, as it's more for object/array values. `toBe()` would be the way to go.
https://jestjs.io/docs/expect#tobevalue
https://jestjs.io/docs/expect#toequalvalue

```suggestion
      expect(actual.field).toBe('all');
```

Reply:
thanks! i didnt know that.  I went ahead and updated our tests where we test primitives to use toBe